### PR TITLE
PR for article on self-perceptions

### PR DESCRIPTION
**Pull request by rinkug**
Author: Rinku Gupta

This is a PR for the curated content article on self perceptions mentioned in issue #348

### CODE REVIEW

**Comment by rinkug** [comment]
This needs to be merged and publish immediately.

File: CuratedContent/SelfPerceptions.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+for their research to highly complex code that will be used on top
+supercomputers of the world. But, while software development skills
+are becoming as integral to science and engineering as much as laboratory
+skills, the community hasn't developed a culture of training for
+the relevant software skills.
+
+The paper surveys close to 150 researchers, encompassing people
+from a rich educational background (with 82% respondents having a
+PhD, which is one of the highest globally recognized degrees for
+students) from the domain of mathematics and statistics, engineering,
```
Comment (bartlettroscoe) [inline CuratedContent/SelfPerceptions.md]: Change "domain" to "domains" here?

File: CuratedContent/SelfPerceptions.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+and computer science. Also relevant is the fact, is that the survey
+tries to analyze how respondents spent their time fractions - whether
```
Comment (bartlettroscoe) [inline CuratedContent/SelfPerceptions.md]: Change to "Also, the survey tries to analyze ..."?

**Comment by rinkug** [comment]
Assigning to @pagrubel to make changes and then publish

**Comment by rinkug** [comment]
This is merged , metadata set and should be published in the next build

File: CuratedContent/SelfPerceptions.md (hunk 1 above)
Comment (rinkug) [reply CuratedContent/SelfPerceptions.md]: > Change "domain" to "domains" here?

Done

File: CuratedContent/SelfPerceptions.md (hunk 2 above)
Comment (rinkug) [reply CuratedContent/SelfPerceptions.md]: Changes done

**Comment by rinkug** [comment]
This is merged , metadata set and should be published in the next build

**Pull request merged**